tools/webpack.config.babel.js

```diff
@@ -1,15 +1,27 @@
 const path = require('path')
 const webpack = require('webpack')
 const AssetsPlugin = require('assets-webpack-plugin')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const { CleanWebpackPlugin } = require('clean-webpack-plugin')
+const CopyPlugin = require('copy-webpack-plugin')
 const babelConfig = require('../.babelrc.js')
 const getVersion = require('git-repo-version')
 
-const isDebug = global.DEBUG === false ? false : !process.argv.includes('--release')
-const useHMR = !!global.HMR // Hot Module Replacement (HMR)
+const createConfig = (env = {}) => {
+  if (env.prod === undefined) {
+    throw Error('You need to specify a mode!')
+  }
+
+  const isDebug = env.prod === 'false'
+  const appConfigName = env.config_name || 'integreat'
+  const appConfig = require(`./${appConfigName}-config`)
```

Comment:
Are there no problems, since these configs are flow-typed?

Reply:
No, you can add `.babel.` to the config name to use babel here.